pkg/bundle/resources.go

```diff
@@ -360,6 +362,10 @@ func readContent(ctx context.Context, progress *progress.Progress, base, name st
 	}
 
 	err = filepath.Walk(temp, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
```

Comment:
```
// The err argument reports an error related to path, signaling that Walk
// will not walk into that directory. The function can decide how to
// handle that error; as described earlier, returning the error will
// cause Walk to stop walking the entire tree.
```
Returning that error will cause walk to abort if one of the paths is not accessible. That changes the existing behavior of the function. If I understand correctly this func is used to read bundle resources, so we probably do want to abort if part of the bundle is unreadable. However we need to document this change.